src/components/BannedCharacters/BannedCharacters.js

```diff
@@ -0,0 +1,66 @@
+import { h } from 'preact';
+import classNames from 'classnames';
+import { DisplayConstants, Colors } from './banned-characters-style-constants';
+
+const BannedCharacters = ({ bannedCharacters = [], highlightedCharacters = [] }) => {
+    if (bannedCharacters.length) {
```

Comment:
maybe in this case early return is better?

Reply:
make sense for readability